src/transformers/models/timm_wrapper/modeling_timm_wrapper.py

```diff
@@ -117,7 +117,15 @@ def __init__(self, config: TimmWrapperConfig):
         super().__init__(config)
         # using num_classes=0 to avoid creating classification head
         extra_init_kwargs = config.model_args or {}
-        self.timm_model = timm.create_model(config.architecture, pretrained=False, num_classes=0, **extra_init_kwargs)
+        try:
+            self.timm_model = timm.create_model(
+                config.architecture, pretrained=False, num_classes=0, **extra_init_kwargs
+            )
+        except RuntimeError as e:
+            raise ImportError(
+                f"You are trying to instantiate `{config.architecture}`, but it's not supported in timm={timm.__version__}. "
+                "Please try updating to the latest timm version with `pip install -U timm`."
+            ) from e
```

Comment:
We should catch an exact exception that the model is unsupported, not a general one